Fix-up minor editorial issues (#272)
ietf-rats-wg/draft-ietf-rats-corim: draft-ietf-rats-corim.md

The following details the security version number (`svn`) and the minimum security version number (`min-svn`) statements.
A security version number is used to track changes to an object (e.g., a secure enclave, a boot loader executable, a configuration file, etc.) that are security relevant.

> Rollback of a security relevant change is considered to be an attack vector and as such, security version numbers cannot be decremented.

Reviewer: ```suggestion
Rollback of a security relevant change is considered to be an attack vector; as such, security version numbers MUST NOT be decremented.
```

Author: I updated the editorial suggestions, but I do not want to add or remove normative wording in this pull request.

> If a security relevant flaw is discovered in the Target Environment and subsequently fixed, the `svn` value is typically incremented.

Reviewer: ```suggestion
If a security relevant flaw is discovered in the Target Environment and is subsequently fixed, incrementing the `svn` value is RECOMMENDED.
```

Author: I updated the editorial suggestions, but I do not want to add or remove normative wording in this pull request.
